<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="7" passed="4" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-03-15T14:34:11Z" name="CFATETESTDATACREATIONs  " finished-at="2021-03-15T14:34:46Z" duration-ms="34866">
    <groups>
    </groups>
    <test started-at="2021-03-15T14:34:11Z" name="callflow" finished-at="2021-03-15T14:34:46Z" duration-ms="34866">
      <class name="com.convirza.cfa_testdata.TestDataHelperClass">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6]" started-at="2021-03-15T19:59:37Z" name="setup" finished-at="2021-03-15T20:04:11Z" duration-ms="274432" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="generateOuthTokenForCompanyAndLocationUser()[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6]" started-at="2021-03-15T20:04:11Z" name="generateOuthTokenForCompanyAndLocationUser" finished-at="2021-03-15T20:04:24Z" duration-ms="12625" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateOuthTokenForCompanyAndLocationUser -->
        <test-method signature="calls()[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6]" started-at="2021-03-15T20:04:24Z" name="calls" finished-at="2021-03-15T20:04:24Z" duration-ms="42" status="FAIL">
          <exception class="java.lang.NumberFormatException">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NumberFormatException: null
	at java.base/java.lang.Integer.parseInt(Integer.java:614)
	at java.base/java.lang.Integer.parseInt(Integer.java:770)
	at com.convirza.cfa_testdata.TestDataHelperClass.calls(TestDataHelperClass.java:120)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NumberFormatException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- calls -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6]" started-at="2021-03-15T20:04:24Z" name="tearDown" finished-at="2021-03-15T20:04:24Z" duration-ms="304" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=calls status=FAILURE method=TestDataHelperClass.calls()[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method signature="campaigns()[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6]" started-at="2021-03-15T20:04:24Z" name="campaigns" finished-at="2021-03-15T20:04:24Z" duration-ms="29" status="FAIL">
          <exception class="java.lang.NumberFormatException">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NumberFormatException: null
	at java.base/java.lang.Integer.parseInt(Integer.java:614)
	at java.base/java.lang.Integer.parseInt(Integer.java:770)
	at com.convirza.cfa_testdata.TestDataHelperClass.campaigns(TestDataHelperClass.java:94)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NumberFormatException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- campaigns -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6]" started-at="2021-03-15T20:04:24Z" name="tearDown" finished-at="2021-03-15T20:04:24Z" duration-ms="100" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=campaigns status=FAILURE method=TestDataHelperClass.campaigns()[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method signature="components()[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6]" started-at="2021-03-15T20:04:24Z" name="components" finished-at="2021-03-15T20:04:26Z" duration-ms="2053" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- components -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6]" started-at="2021-03-15T20:04:26Z" name="tearDown" finished-at="2021-03-15T20:04:26Z" duration-ms="55" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=components status=SUCCESS method=TestDataHelperClass.components()[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method signature="groups()[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6]" started-at="2021-03-15T20:04:26Z" name="groups" finished-at="2021-03-15T20:04:30Z" duration-ms="4092" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- groups -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6]" started-at="2021-03-15T20:04:30Z" name="tearDown" finished-at="2021-03-15T20:04:30Z" duration-ms="134" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=groups status=SUCCESS method=TestDataHelperClass.groups()[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method signature="numbers()[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6]" started-at="2021-03-15T20:04:30Z" name="numbers" finished-at="2021-03-15T20:04:41Z" duration-ms="10525" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- numbers -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6]" started-at="2021-03-15T20:04:41Z" name="tearDown" finished-at="2021-03-15T20:04:41Z" duration-ms="89" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=numbers status=SUCCESS method=TestDataHelperClass.numbers()[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method signature="trackingNumbers()[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6]" started-at="2021-03-15T20:04:41Z" name="trackingNumbers" finished-at="2021-03-15T20:04:41Z" duration-ms="2" status="FAIL">
          <exception class="java.lang.NumberFormatException">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NumberFormatException: null
	at java.base/java.lang.Integer.parseInt(Integer.java:614)
	at java.base/java.lang.Integer.parseInt(Integer.java:770)
	at com.convirza.cfa_testdata.TestDataHelperClass.trackingNumbers(TestDataHelperClass.java:107)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NumberFormatException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- trackingNumbers -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6]" started-at="2021-03-15T20:04:41Z" name="tearDown" finished-at="2021-03-15T20:04:41Z" duration-ms="55" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=trackingNumbers status=FAILURE method=TestDataHelperClass.trackingNumbers()[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method signature="webhooks()[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6]" started-at="2021-03-15T20:04:41Z" name="webhooks" finished-at="2021-03-15T20:04:46Z" duration-ms="4666" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- webhooks -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6]" started-at="2021-03-15T20:04:46Z" name="tearDown" finished-at="2021-03-15T20:04:46Z" duration-ms="51" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=webhooks status=SUCCESS method=TestDataHelperClass.webhooks()[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="send_report()[pri:0, instance:com.convirza.cfa_testdata.TestDataHelperClass@52719fb6]" started-at="2021-03-15T20:04:46Z" name="send_report" finished-at="2021-03-15T20:05:03Z" duration-ms="16740" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- send_report -->
      </class> <!-- com.convirza.cfa_testdata.TestDataHelperClass -->
    </test> <!-- callflow -->
  </suite> <!-- CFATETESTDATACREATIONs   -->
</testng-results>
